abios_sdk.go

```diff
@@ -96,9 +97,9 @@ func (a *client) authenticator() {
 }
 
 // NewAbios returns a new endpoint-wrapper for api version 2 with given credentials.
-func New(username, password string) *client {
+func New(username, password, url string) *client {
```

Comment:
Should not be parameter to `New`. Doing it like this requires *everyone* to *always* specify a url.

I think the most sensible would be to have `New` use the default `baseUrl` (which goes to prod) and creating a new constructor where you can specify url.